scripts/revert_upgrade_voting_delegation_holesky.py

```diff
@@ -0,0 +1,92 @@
+"""
+Revert Voting Delegation Upgrade
+
+1. Push new Voting app version to the Voting Repo 0x2997EA0D07D79038D83Cb04b3BB9A2Bc512E3fDA
+2. Downgrade the Aragon Voting contract implementation to 0xcB738a79baeA44C93Ee46c02EF0FA975Bc4d058f
+3. Downgrade TRP voting adapter to 0x1dF997832b44b7ED00597f103165920537c980D4
+
+"""
+import time
+
+from typing import Dict
+from brownie.network.transaction import TransactionReceipt
+from utils.voting import bake_vote_items, confirm_vote_script, create_vote
+from utils.ipfs import upload_vote_ipfs_description, calculate_vote_ipfs_description
+from utils.config import (
+    get_deployer_account,
+    get_is_live,
+    get_priority_fee,
+    contracts,
+)
+from utils.repo import add_implementation_to_voting_app_repo
+from utils.kernel import update_app_implementation
+from utils.agent import agent_forward
+
+downgraded_trp_voting_adapter = "0x1dF997832b44b7ED00597f103165920537c980D4"
+
+downgraded_voting_app = {
+    "address": "0xcB738a79baeA44C93Ee46c02EF0FA975Bc4d058f",
+    "content_uri": "0x",
+    "id": "0x0abcd104777321a82b010357f20887d61247493d89d2e987ff57bcecbde00e1e",
+    "version": (4, 0, 0),
+}
+
+description = """
+Voting Delegation downgrade Holesky
+"""
+
+
+def start_vote(tx_params: Dict[str, str], silent: bool) -> bool | list[int | TransactionReceipt | None]:
```

Comment:
Similar to the previous Voting upgrade script, it would be nice to have a test for the revert script (see test_revert_vote_2022_06_21)

Reply:
Fixed in 98c41312c4dcbf45286acc799ee16c3e46be578a.